src/behaviors.js

```diff
@@ -47,7 +47,7 @@ Marionette.Behaviors = (function(Marionette, _) {
         behaviorEvents = Marionette.normalizeUIKeys(behaviorEvents, getBehaviorsUI(b));
 
         var j = 0;
-        _.each(behaviorEvents, function(behaviour, key) {
+        _.each(behaviorEvents, function(behaviorEvent, key) {
```

Comment:
Yeah sounds good to me :)

On Wednesday, 5 August 2015, W. van Kuipers <notifications@github.com>
wrote:

> In src/behaviors.js
> <https://github.com/marionettejs/backbone.marionette/pull/2687#discussion_r36284589>
> :
>
> > @@ -47,7 +47,7 @@ Marionette.Behaviors = (function(Marionette, _) {
> >          behaviorEvents = Marionette.normalizeUIKeys(behaviorEvents, getBehaviorsUI(b));
> >
> >          var j = 0;
> > -        _.each(behaviorEvents, function(behaviour, key) {
> > +        _.each(behaviorEvents, function(behaviorEvent, key) {
>
> Noted, maybe eventHandler is the right therm then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marionettejs/backbone.marionette/pull/2687/files#r36284589>
> .
>